package.json

```diff
@@ -13,6 +13,24 @@
     "@oclif/plugin-help": "^3",
     "tslib": "^1"
   },
+
+  
+    "jspm": {
```

Comment:
@noa-moshe , can we move this to its own sample file? It has no place in the package.json of this project as we don't use these dependencies :)